ajax/libs/vega/package.json

```diff
@@ -13,8 +13,13 @@
     {
       "basePath": "",
       "files": [
-        "vega.js",
-        "vega.min.js"
+        "vega*.(js|css|map)"
+      ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "vega*.(js|css|map)"
```

Comment:
Oh okay. I can change it but with the basePath set to build I suspected this would be sufficient for now and into the future.